src/components/Item/Item.test.js

```diff
@@ -7,7 +7,6 @@ import { testItem } from '../../test-utils'
 import { Item } from './Item'
 
 jest.mock('../../data/maps')
-jest.mock('../../data/shop-inventory')
```

Comment:
@jeremyckahn is there a specific reason you chose to mock this? it seems like a good thing to test the real data here to me. though, the reason i looked is because the tests all fail with the mock in place for the new `itemIds` export. let me know what you think

Reply:
Overall I wanted to keep game data/content reasonably decoupled from the logic for the tests. Some of the test setup is fragile and less than ideal, admittedly. I'd be fine with removing this mock if that's what's works for you here!

I don't love the the test situation for Farmhand right now, and I'm open to new ideas and approaches!

Reply:
alright, I spend lots of time thinking about unit testing so I'll take a closer look at some things and make some PRs to suggest changes